channels-rke2.yaml

```diff
@@ -116,3 +116,9 @@ releases:
       rancher-vsphere-csi:
         repo: rancher-charts
         version: 100.0.0
+  - version: v1.21.5+rke2r1
```

Comment:
Have we confirmed that the charts packaged in v1.21.5 will be the same?

How is this going to be different from what is being introduced in dev-v.6 (aka Rancher 2.6.1) 

Will there be a v1.21.5+rke2r2 shipping with 2.6.1? 

https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.6/channels-rke2.yaml#L119

Reply:
I'm also confused - why is this going into this specific target branch, and why is it missing all the stuff that's currently in dev-v2.6?

Reply:
It's on a separate target branch as we wanted to get these k8s versions out and available due to the security fixes in it and not wait until 2.6.1 for the updates to be released. RKE1 was already released from this branch into production.

Reply:
Due to the lack of support in Harvester, the min Rancher version will need be changed to 2.6.1 as well.